app/controllers/reviewers/applications_controller.rb

```diff
@@ -79,5 +80,9 @@ def applications_table
       }
       Selection::Table.new(applications: Application.rateable, options: options)
     end
+
+    def grab_mentor_comments
```

Comment:
I'd rather have this as a lazily loaded helper method. This would remove the somewhat procedural statement in line 31 and the call to an instance variable in the template view.

```ruby
# in the controller

…

private

def mentor_comments
  @mentor_comments ||= Mentor::Comment.where(commentable_id: @application.id)
end
helper_method :mentor_comments # or up top, don't know if we have a style guide for this /shrug
```